addon/mixins/route.js

```diff
@@ -193,6 +199,7 @@ const RouteMixin = Ember.Mixin.create({
       currentPage: startingPage,
       _firstPageLoaded: false,
       _perPage: perPage,
+      _store: store,
```

Comment:
> What are your thoughts on this? Does your custom store use the same query method?

My thoughts are that a custom store, just like a custom route or component, extends the Ember default (in this case Store) and as such is supposed to implement the same behavior as Ember.DS.

I don't see a use case where one would want to customuze the `_storeFindMethod`.